pages/utils/google-analytics/ga-track.js

```diff
@@ -0,0 +1,34 @@
+import ReactGA from 'react-ga';
+
+const isProd = true; //TODO - Track only production analytics otherwise dev and test would be tracked as well
```

Comment:
Need to figure out how to differentiate between different environments

Reply:
I should be able to add a filter on GA that will let me differentiate traffic from test.affordableseattle.org and affordableseattle.org.

Reply:
ok, i'll remove the that isProd for now and you can play around with the metrics.